sdk/nodejs/runtime/settings.ts

```diff
@@ -598,6 +600,13 @@ export function rpcKeepAlive(): () => void {
     return done!;
 }
 
+/**
+ * Returns if the engine supports package references.
```

Comment:
Again I think we should be consistent about the fact that this flag includes parameterization